esphome/components/atm90e32/text_sensor/__init__.py

```diff
@@ -0,0 +1,64 @@
+import esphome.codegen as cg
+from esphome.components import text_sensor
+import esphome.config_validation as cv
+from esphome.const import CONF_ID, CONF_PHASE_A, CONF_PHASE_B, CONF_PHASE_C
+
+from .. import atm90e32_ns
+from ..sensor import ATM90E32Component
+
+ATM90E32PhaseStatusSensor = atm90e32_ns.class_(
+    "ATM90E32PhaseStatusSensor",
+    text_sensor.TextSensor,
+    cg.Parented.template(ATM90E32Component),
+)
+
+ATM90E32FreqStatusSensor = atm90e32_ns.class_(
+    "ATM90E32FreqStatusSensor",
+    text_sensor.TextSensor,
+    cg.Parented.template(ATM90E32Component),
+)
+
+CONF_PHASE_STATUS = "phase_status"
+CONF_FREQ_STATUS = "freq_status"
```

Comment:
Longer, proper names are preferred
```suggestion
CONF_FREQUENCY_STATUS = "frequency_status"
```